### PR TITLE
Adds  miner's winter coats to mining and icebox station gulag

### PR DESCRIPTION
**Pull request by Ikoden**
Four winter coats spawn in ice box station's gulag and each miner's locker has a mining coat inside.

🆑
Four winter coats spawn in ice box station's gulag and each miner's locker has a mining coat inside.
/🆑

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
add winter coats here instead of mapping them 
https://github.com/tgstation/tgstation/blob/add74bee56cfde91602eb8ee28e3cae5959bc4f6/code/modules/mining/mine_items.dm#L49

**Comment by Ikoden** [comment]
> 
> 
> add winter coats here instead of mapping them
> 
> https://github.com/tgstation/tgstation/blob/add74bee56cfde91602eb8ee28e3cae5959bc4f6/code/modules/mining/mine_items.dm#L49

It's about gulag (which i forgot to mention), not the miners.

**Comment by trollbreeder** [comment]
Miner lockers dont seem to spawn with wintercoats, add some mining wintercoats there (or cargo wintercoats if you cant find mining ones)

**Comment by Ikoden** [comment]
I added winter coat to the mine_items.dm

**Comment by Ikoden** [comment]
I changed the winter coat in mine_items.dm to a miner's winter coat.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
file locations

**Comment by Ikoden** [comment]
What do you mean by file locations? Please explain it simply because IM ULTRA NOOB AT CODING AND GITHUB

**Comment by ExcessiveUseOfCobblestone** [comment]
if you click files changed you'll notice the entire file is new. this is because you put them under tgstation instead of their appropriate locations (code/maps/... and code/modules/mining/....).

This is a result of trying to use the webclient to make changes. You should use a program like gitkraken to stage the changes.

**Comment by Ikoden** [comment]
Okay, I will work on that.

**Comment by Ikoden** [comment]
Is it good now?

**Comment by Ikoden** [comment]
There are some conflicts, im working on it. Give me some time.

**Comment by Ikoden** [comment]
Having problem with the map change. Will work on it.

**Comment by Ikoden** [comment]
OK. I used the guide to map change and all should work now.

**Comment by Ikoden** [comment]
So like, what's going with merge conflict? Do i need to do something ?

**Comment by Ikoden** [comment]
Okay, I added the wintercoats via code change and not by using map editor. They should spawn on the racks in gulag. It will either spawn 2 or 4. I'm not sure.

**Comment by Ikoden** [comment]
Everything should work now. I checked it. Four winter coats spawn in ice box station's gulag and each miner's locker has a mining coat inside.

**Comment by spookydonut** [comment]
should winter coats spawn on all maps or only on icebox?

**Comment by Ikoden** [comment]
They probably spawn in miner's locker on all maps. The question is - should they?  I can see reason both for pro and against

**Comment by ExcessiveUseOfCobblestone** [comment]
it doesn't have special values does it?

if it's only vanity changes compared to other winter coats then it's not an issue.

**Comment by Ikoden** [comment]
Miner's coat has, the normal version has bio 10 afaik
armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)

**Comment by ExcessiveUseOfCobblestone** [comment]
armor is a fiesta and it's only 10% difference so 🤷

**Comment by Ikoden** [comment]
Okie dokie. Can you guys review it before icebox gets changed again, please?

File: IcemoonUnderground_Above.dmm [hunk 1]
```diff
@@ -0,0 +1,732 @@
+"aa" = (/obj/docking_port/stationary{dir = 8; dwidth = 2; height = 5; id = "laborcamp_away"; name = "labor camp"; width = 9},/turf/closed/mineral/random/snow,/area/icemoon/underground/unexplored/rivers)
```
Comment (spookydonut) [inline IcemoonUnderground_Above.dmm:1]: delete this file

**Review by spookydonut — CHANGES_REQUESTED** [review]
delete dis

File: IcemoonUnderground_Above.dmm (hunk 1 above)
Comment (Ikoden) [reply IcemoonUnderground_Above.dmm:1]: Why delete this file? This file is needed to give winter coat to gulag.

**Comment by Ikoden** [comment]
Why do you want me to delete IcemoonUnderground_Above.dmm  while it's needed to give winter coats to gulag prisoners?

File: IcemoonUnderground_Above.dmm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply IcemoonUnderground_Above.dmm:1]: note the file location and the fact you have 2 same-named files in the files changed tab.

**Pull request closed without merging**